docs/rules/explicit-length-check.md

```diff
@@ -10,21 +10,21 @@ This rule is partly fixable.
 if (string.length) {}
 if (array.length) {}
 if (!array.length) {}
+if (array.length !== 0) {}
 ```
 
 ### Pass
 
 ```js
 if (string.length > 0) {}
 if (array.length > 0) {}
-if (array.length !== 0) {}
 if (array.length === 0) {}
 ```
 
 
 ## Zero comparisons
 
-Enforce comparison with `!== 0` when checking for zero length.
+Enforce comparison with `=== 0` when checking for zero length.
```

Comment:
I believe this was a typo because this section talks about zero comparisons. So I fixed it.